### PR TITLE
fixes #29: add user and list members on the main page

### LINKED ISSUE #29: Members list on the main page
When connecting to the main page, should show the list of members belonging to the organization.

Should replace: http://www.coderbunker.com/members

### PR DESCRIPTION
**Pull request by rngadam**
display-member
lists-users
add-member

### CODE REVIEW

**Comment by thimovss** [comment]
Is display of name needed, if yes it is not centered/looks out of position
fireshot screen capture 004 - membership management system - localhost_3000_log

File: app/client/templates/admin.js [hunk 1]
```diff
@@ -3,4 +3,31 @@ Template.admin.events({
...
+              },
+              onEdit: function(params) {
+                console.log(params);
+                console.log('onEdit ' + params.ids);
+                FlowRouter.go('/users/' + params.ids[0] + '/edit');
+              },
+
+              collection: Meteor.users,
+              rowsPerPage: 10,
+              showFilter: true,
```
Comment (thimovss) [inline app/client/templates/admin.js:23]: Needs checking if all fields are entered correctly, you can add blank users

**Comment by thimovss** [comment]
There is no way to see the list if you are logged in ( maybe make the CoderBunker at the top left a link)

**Comment by thimovss** [comment]
Maybe add a blank/filler image so the name/title/bias stay aligned

**Comment by thimovss** [comment]
If you flush DB it does not fetch user info again

**Comment by rngadam** [comment]
- [x] Needs checking if all fields are entered correctly, you can add blank users
- [x] There is no way to see the list if you are logged in ( maybe make the CoderBunker at the top left a link)
- [x] If you flush DB it does not fetch user info again
- [x] Maybe add a blank/filler image so the name/title/bias stay aligned

**Comment by rngadam** [comment]
- name and email now required
- Members menu option
- added user backup
- always display a profile image of some sort

File: README.md [hunk 2]
```diff
@@ -35,3 +35,13 @@ meteor run --settings ../secrets/settings/localhost-3000.json
 ### Deployment
 
 see .deploy/README.md
+
+
+## Backup
+
```
Comment (thimovss) [inline README.md:41]: 2 times ## backup?

File: app/client/templates/user_update.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      console.log('onError');
+      console.log(error);
+    }
+  }
+});
+
+Template.user_update.helpers({
+    user: function() {
+      var u = Meteor.users.findOne(FlowRouter.getParam('_id'));
+      return u;
```
Comment (thimovss) [inline app/client/templates/user_update.js]: no need to store u locally

File: backup.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+mongodump -h 127.0.0.1 --port 3001 -d meteor --collection users --out ./secrets/dump
```
Comment (thimovss) [inline backup.sh:2]: Meteor uses port N, N+1 and N+2, so use port 3004 instead.

**Comment by thimovss** [comment]
maybe add secrets/dump/meteor/ to .gitignore

File: README.md (hunk 2 above)
Comment (rngadam) [reply README.md:41]: fixed

File: app/client/templates/user_update.js (hunk 3 above)
Comment (rngadam) [reply app/client/templates/user_update.js]: fixed

File: backup.sh (hunk 4 above)
Comment (rngadam) [reply backup.sh:2]: I'm connecting to the existing MongoDB... run by Meteor... to dump DB

**Comment by rngadam** [comment]
on purpose: keeping the backups encrypted in the repo for the users so we can restore later

**Pull request merged**